src/patternfly/components/Pagination/pagination.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../patternfly-imports";
+
+.pf-c-pagination {
+  // Base
+  --pf-c-pagination--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-pagination--child--MarginRight: var(--pf-global--spacer--lg);
+
+  // Dropdown
+  --pf-c-pagination--c-dropdown__toggle--Color: var(--pf-global--Color--100);
+  --pf-c-pagination__menu-text--Color: var(--pf-global--Color--200);
```

Comment:
Is this also referring to the options-menu component? Or does this refer to pagination. Otherwise update naming.

Reply:
are you suggesting change this to like `--pf-c-pagination--c-options-menu__menu-text--Color`? If so I don't think we need to do that since this class has it's own BEM name. If I were just targeting text inside the options menu and the element didn't have it's own pagination BEM name, I would write the var like that.